content/academy/apify_platform/deploying.md

```diff
@@ -12,6 +12,35 @@ This section will discuss how to use your newfound knowledge of the Apify platfo
 
 Because actors are basically just chunks of code running in Docker containers, you're able to **_Actorify_** just about anything!
 
+## Our example project
+
+For this section, we'll be turning our example Node.js project into an actor. The example project looks like this:
+
+```JavaScript
+// index.js
+const addAllNumbers = (...nums) => nums.reduce((total, curr) => (total += curr));
+
+console.log(addAllNumbers(1, 2, 3, 4)) // -> 10
+```
```

Comment:
I would suggest to changing this to our template solution for examples which what the Rust code uses. E.g. Read URL from input, get HTML from the URL and save it to output.
1. It is a real-ish use-case
2. It is about scraping
3. We can follow this use-case in all our non-JS examples
4. You can use my Rust example as-is

Reply:
@mnmkng What do you think?

I think that the most important thing is that they understand Dockerfile, CLI, Inputs & outputs, and Github integration. That is why I provided such a simple example that you don't even have to think about to understand. This section doesn't have to be related to scraping, and I would argue shouldn't have anything about scraping in it.

Reply:
I don't think it needs to be about scraping, but i think the examples should use the Apify SDK as much as possible, so that it gets ingrained into the users' brains :)

PS: Don't forget that we're splitting Apify SDK into Apify SDK and Crawlee, so Apify SDK will only have the platform controlling functions like reading input, saving to storages etc. No crawling.

Reply:
@mnmkng So, would you say to change the example right now, or wait for the new library before changing it?

Reply:
The Apify SDK will have the same top level commands like getInput or pushData. So just avoid the scraping part and you won't have to change anything later.